plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -313,6 +310,25 @@ export class ExpressInstrumentation extends InstrumentationBase {
           }
         }
       };
+
+      // `handle` isn't just a regular function in some cases. It also contains
+      // some properties holding metadata and state so we need to proxy them
+      // through through patched function
+      // ref: https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1950
+      Object.keys(original).forEach(key => {
+        Object.defineProperty(patched, key, {
```

Comment:
I wonder why define a property with getter and setter.

Will this work as well?

```js
Object.keys(original).forEach(key => patched[key] = original[key])
```

Or

```js
Object.entries(original).forEach([key, value] => patched[key] = value)
```

Reply:
Saw your comment on why this might not work. Thanks for the explanation 🙏 makes sense now